docs/development/README.adoc

```diff
@@ -7,26 +7,78 @@
 
 = T developer documentation
 
-== xref:local-t-network.adoc[Setting up local T network]
+== Quick installation
 
-For development purposes, it is convenient to set up a local Ethereum client with
-a group of connected peers. A note-taken walkthrough covers the entire process
+=== Pre-requirements
+For development purposes, it is convenient to set up a local Ethereum client with a group of connected peers. A note-taken walkthrough link:local-t-network.adoc[set up local T network] covers the entire process
 including setting up a local Ethereum client, deploying T contracts and
-configuring T peers.
+configuring T peers. +
 
-== Quick installation
+Additionaly, to start a client we need to provide config options either by creating a config file or passing the required parameters. For simple installation using a config file please refer to a <<sample_config_file,sample configuration>> file that is placed under `/configs/` dir.
```

Comment:
We should add a paragraph that it is required to create configuration files for each instance one wants to run and place them inside the `configs/` directory.